lib/chef/application/windows_service.rb

```diff
@@ -264,6 +266,14 @@ def auto_log_level?
         Chef::Config[:log_level] == :auto
       end
 
+      def resolve_log_location
+        if Chef::Config[:log_location] && Chef::Config[:log_location].is_a?(String)
```

Comment:
Then it logs to `DEFAULT_LOG_LOCATION` since `STDOUT` fails the `.is_a?(String)` test. I previously was just checking if it was equal to `STDOUT` but the unit tests were fiddling with `STDOUT` and so I just want to make sure it is not an instance of IO.